packages/runtime-common/realm.ts

```diff
@@ -804,6 +805,36 @@ export class Realm {
     });
   }
 
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  private async realmInfo(_request: Request): Promise<Response> {
+    let fileURL = this.paths.fileURL(`.realm.json`);
+    let localPath: LocalPath = this.paths.local(fileURL);
+    let realmConfig = await this.readFileAsText(localPath);
+    let name = 'Unnamed Workspace';
+    if (realmConfig) {
+      try {
+        let realmConfigJson = JSON.parse(realmConfig.content);
+        if (realmConfigJson.name) {
+          name = realmConfigJson.name;
+        }
+      } catch (e) {
+        this.#log.warn(`failed to parse realm config: ${e}`);
+      }
+    }
+    let doc = {
+      data: {
+        id: this.paths.url.toString(),
+        type: 'realm-info',
+        attributes: {
+          name,
+        },
+      },
+    };
+    return createResponse(JSON.stringify(doc, null, 2), {
+      headers: { 'content-type': SupportedMimeType.CardJson },
```

Comment:
this is not card json--card json has the `meta.adoptsFrom` (notably it should pass this assertion test https://github.com/cardstack/boxel/blob/main/packages/runtime-common/card-document.ts#L187-L189). rather, this is just normal application/vnd.api+json